app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self) -> None:
```

Comment:
Code Efficiency: It's a good practice to use properties for encapsulation in Python. Instead of creating a method `eat_favourite_dish`, you could create a property `favourite_dish` and a corresponding setter. Then you can simply call `dwarf.favourite_dish` to get the favourite dish and `dwarf.favourite_dish = 'new dish'` to set a new favourite dish.